http/handler_store.go

```diff
@@ -22,6 +22,11 @@ import (
 
 type storeHandler struct{}
 
+type CollectionDefinition struct {
```

Comment:
note: whatever solution is for identical struct in `cli/collection_describe.go` that we discussed should also apply here I believe